docs/ExtensionPoints/SubscriptionManagement/README.md

```diff
@@ -0,0 +1,119 @@
+# Subscription Management Extension
+
+This extension renders in the Product Details and Product Variants Page inside a Modal container.
+
+## Extension Points
+
+Subscription Management has multiple extension points. Each is triggered by a different merchant action, receives different data, and has a different responsibility.
+
+The different extension points are:
+
+- SubscriptionManagementAdd
+- SubscriptionManagementCreate
+- SubscriptionManagementEdit
+- SubscriptionManagementRemove
+
+They must be rendered separately.
+
+```js
+function Add(root, api) {
+  root.appendChild(root.createText('Hello, World'));
+  root.mount();
+}
+function Create() {
+  /* ... */
+}
+function Edit() {
+  /* ... */
+}
+function Remove() {
+  /* ... */
+}
+
+render(ExtensionPoint.SubscriptionManagementAdd, Add);
+render(ExtensionPoint.SubscriptionManagementCreate, Create);
+render(ExtensionPoint.SubscriptionManagementEdit, Edit);
+render(ExtensionPoint.SubscriptionManagementRemove, Remove);
+```
+
+### Data API
+
+See [Data](../../Utilities/Data.md) for more info about how to access subscription data provided by the host page.
```

Comment:
Data documentation is missing (there are `ProductData` docs)

Reply:
`ProductData` was changed to `Data` - I've removed it